ReactiveCocoa/UIKit/UIGestureRecognizer.swift

```diff
@@ -0,0 +1,25 @@
+import ReactiveSwift
+import UIKit
+import enum Result.NoError
+
+extension Reactive where Base: UIGestureRecognizer {
+	
+	/// Create a signal which sends a `next` event for each gesture event
+	///
+	/// - returns:
+	///   A trigger signal.
+	
+	public var recognised: Signal<(), NoError> {
```

Comment:
This should be `recognized` with a _z_ if the API doesn't change. The project uses US English, but also that's how `UIGestureRecognizer` spells it. :relaxed: